packages/workspace-tools/src/generators/create-enum/templates/__outputName__

```diff
@@ -0,0 +1 @@
+<%- content %>
```

Comment:
Do we need this?

Reply:
Yes, Nx needs an `ejs` template in order to generate files 🤷‍♂️